library/aes.c

```diff
@@ -222,7 +222,6 @@ static const uint32_t FT3[256] = { FT };
 
 #undef FT
 
-#if !defined(MBEDTLS_AES_ONLY_ENCRYPT)
```

Comment:
I don't think this line should be removed. This looks like a rebase mistake. Actually all preprocessor directives related to this option seem to have disappeared from this file (except those in the self-test function).

Reply:
yeah, that's weird, I did not touch those manually. Git rebase magic... I'll try to fix.